.github/workflows/check_pr.yml

```diff
@@ -62,6 +64,17 @@ jobs:
       - uses: actions/checkout@v2
       - uses: dart-lang/setup-dart@v1.3
 
+      - name: Install sudo package
+        run: apt update && apt install sudo
```

Comment:
Why is it needed?

Reply:
It is added since the solana-local-validator gh action uses sudo access